boxes/npx.js

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env node
+import { Command } from "commander";
+import select from "@inquirer/select";
+import input from "@inquirer/input";
+import confirm from "@inquirer/confirm";
+const program = new Command();
+import tiged from "tiged";
+import { exec, execSync } from "child_process";
+import pty from "node-pty";
+import path from "path";
+import os from "os";
+import fs from "fs";
+import chalk from "chalk";
+
+const { log, warn, info } = console;
+const targetDir = path.join(os.homedir(), ".aztec/bin"); // Use os.homedir() to get $HOME
+
+function updatePathEnvVar() {
+  // Detect the user's shell profile file based on common shells and environment variables
+  const homeDir = os.homedir();
+  let shellProfile;
+  if (process.env.SHELL?.includes("bash")) {
+    shellProfile = path.join(homeDir, ".bashrc");
+  } else if (process.env.SHELL?.includes("zsh")) {
+    shellProfile = path.join(homeDir, ".zshrc");
+  } else {
+    // Extend with more conditions for other shells if necessary
+    warn("Unsupported shell or shell not detected.");
+    return;
+  }
+
+  // Read the current content of the shell profile to check if the path is already included
+  const profileContent = fs.readFileSync(shellProfile, "utf8");
+  if (profileContent.includes(targetDir)) {
+    log(`${targetDir} is already in PATH.`);
+    return;
+  }
+
+  // Append the export command to the shell profile file
+  const exportCmd = `\nexport PATH="$PATH:${targetDir}" # Added by Node.js script\n`;
+  fs.appendFileSync(shellProfile, exportCmd);
+
+  info(`Added ${targetDir} to PATH in ${shellProfile}.`);
+}
+
+program.action(async () => {
```

Comment:
Given we have a single action with no params, do we really need Commander here?

Reply:
Hmm good point, I did it as I was expecting more actions to be needed. This will probably be the case (i.e. for running the sandbox *only*) and seems minimal overhead since the user is not using this dependency, so I'll leave it if you don't mind!